distribution/springboot_setup/sender/routes/out-bound-lims-route.xml

```diff
@@ -0,0 +1,41 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+    <route id="out-bound-lims" errorHandlerRef="deadLetterChannelBuilder">
+        <from uri="direct:out-bound-lims"/>
+        <log message="Processing db event: ${body}" />
+
+        <unmarshal>
+            <json library="Jackson" unmarshalTypeName="org.openmrs.eip.component.entity.DbEvent" />
+        </unmarshal>
+        <choice>
+            <when>
+                <jsonpath>
+                    $.[?(@.entityTableName=='test_order') ]
+                </jsonpath>
+                <setProperty name="lab-order-uuid">
+                    <simple>${body.entityId}</simple>
+                </setProperty>                
+                <setBody>
+                    <simple>{{openmrs.username}}:{{openmrs.password}}</simple>
```

Comment:
That's ok for now, but perhaps worth a new EIP ticket to frame the work about nailing OpenMRS authentications through a reusable pattern for all routes.